spec/index.bs

```diff
@@ -1888,6 +1889,8 @@ interface GPUAdapterInfo {
     readonly attribute DOMString architecture;
     readonly attribute DOMString device;
     readonly attribute DOMString description;
+    readonly attribute unsigned long subgroupMinSize;
```

Comment:
@alan-baker @dneto0 is there a reason we are adding these to `GPUAdapterInfo` instead of `GPUSupportedLimits`? It seems to fit within the existing limits but maybe there is a reason not to do that?

Reply:
They seem like limits, but aren't really. There is no clear definition of better for either value. Also the user cannot request different values than the hardware defaults. So putting it in adapter info makes it clearer that is a just a property of the hardware.

Reply:
See #4834  where this is discussed more

Reply:
The last thing Kai said there was:

> Yes, I agree we should add properties (as a separate concept from limits) as soon as we need them.

So, now we need them, right? Is just adding attributes to `GPUAdapterInfo` the way we want to handle properties going forward?

Reply:
> So, now we need them, right?

Yes.

>  Is just adding attributes to GPUAdapterInfo the way we want to handle properties going forward?

Yes, I think so.  In the IDL it's "just" another property name. 

(for native APIs, outside scope here(!), use a chained struct)

Reply:
At the F2F we decided they would go somewhere inside GPUAdapterInfo. On one of the PRs we considered whether it should be a subobject, but I think it makes sense to just add them directly.